pkg/services/sqlstore/org_users.go

```diff
@@ -40,8 +49,27 @@ func AddOrgUser(cmd *m.AddOrgUserCommand) error {
 			Updated: time.Now(),
 		}
 
-		_, err := sess.Insert(&entity)
-		return err
+		_, err = sess.Insert(&entity)
+		if err != nil {
+			return err
+		}
+
+		userOrgs := make([]*m.UserOrgDTO, 0)
```

Comment:
if the size is 0 there is no point using `make()`. 

https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Reply:
Good point! Fixed